include/measurement_kit/ooni/nettests.hpp

```diff
@@ -36,6 +36,11 @@ void http_header_field_manipulation(std::string input, Settings,
                                     Var<Reactor> = Reactor::global(),
                                     Var<Logger> = Logger::global());
 
+void facebook_messenger(std::string input, Settings,
```

Comment:
If I'm correct, this test does not take any input. As such, you don't need to pass input here, @joelanders.

Reply:
@joelanders can you remove the `input` parameter from here, since the test does not receive any input?

Reply:
done